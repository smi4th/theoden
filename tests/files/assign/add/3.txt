a=5.0 + 10.0;