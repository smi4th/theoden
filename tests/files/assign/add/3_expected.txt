---TEST OUTPUT---
---MEMORY---
a = ('float', 15.0)
---FUNCTIONS---
